packages/alea-frontend/pages/acl/new.tsx

```diff
@@ -0,0 +1,200 @@
+  const [updaterACLId, setUpdaterACLId] = useState<string | ''>('');
+  const [isOpen, setIsOpen] = useState(false);
+  const [tempMemberUserId, setTempMemberUserId] = useState<string>('');
+  const [tempMemberACL, setTempMemberACL] = useState<string>('');
+  const router = useRouter();
+
+  const handleAddMemberId = (event: React.KeyboardEvent<HTMLInputElement> | React.MouseEvent<HTMLElement>) => {
+    if ((event.type === 'keydown' && (event as React.KeyboardEvent).key === 'Enter') || event.type === 'click') {
+      if (tempMemberUserId) {
+        setMemberUserIds([...memberUserIds, tempMemberUserId]);
+        setTempMemberUserId('');
+      }
+    }
+  };
+
+  const handleRemoveMemberId = (idToRemove: string) => {
+    setMemberUserIds(memberUserIds.filter((id) => id !== idToRemove));
+  };
+
+  const handleAddMemberACL = (event: React.KeyboardEvent<HTMLInputElement> | React.MouseEvent<HTMLElement>) => {
+    if ((event.type === 'keydown' && (event as React.KeyboardEvent).key === 'Enter') || event.type === 'click') {
+      if (tempMemberACL) {
+        setMemberACLIds([...memberACLIds, tempMemberACL]);
+        setTempMemberACL('');
+      }
+    }
+  };
+
+  const handleRemoveMemberACL = (aclToRemove: string) => {
+    setMemberACLIds(memberACLIds.filter((acl) => acl !== aclToRemove));
+  };
+
+  const handleSubmit = async () => {
+    const newAcl: CreateACLRequest = {
+      id: aclId,
+      description,
+      memberUserIds,
+      memberACLIds,
+      updaterACLId,
+      isOpen,
+    };
+    console.log('New ACL:', newAcl);
+    try {
+      await createAcl(newAcl);
+      router.replace(`/acl/${aclId}`);
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  return (
+    <MainLayout>
+      <Box
+        sx={{
+          m: '0 auto',
+          maxWidth: '800px',
+          p: '10px',
+          width: '100%',
+          boxSizing: 'border-box',
+        }}
+      >
+        <Typography fontSize={24} m="10px 0px">
+          Create ACL
+        </Typography>
+        <TextField
+          label="ACL ID"
+          variant="outlined"
+          value={aclId}
+          onChange={(e) => setAclId(e.target.value)}
+          size="small"
+          sx={{ mb: '20px' }}
+          fullWidth
+        />
+        <TextField
+          label="Description"
+          variant="outlined"
+          value={description}
+          onChange={(e) => setDescription(e.target.value)}
+          size="small"
+          sx={{ mb: '20px' }}
+          fullWidth
+        />
+        <Box mb="20px">
+          <TextField
+            label="Add Member ID"
+            variant="outlined"
+            size="small"
+            value={tempMemberUserId}
+            onChange={(e) => setTempMemberUserId(e.target.value)}
+            onKeyDown={handleAddMemberId}
+            sx={{ mb: '10px' }}
+            fullWidth
+            InputProps={{
+              startAdornment: (
+                <InputAdornment position="start">
```

Comment:
does this need `cursor: pointer` css property?

Reply:
we don't need to add it.